website/docs/components/table/advanced-table/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,42 @@
+## Anatomy
+
+### Advanced Table headers
+
+![](/assets/components/table/advanced-table/advanced-table-header-select-anatomy.png)
+
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Checkbox         | Optional, but required when cells yield a checkbox |
+| Label            | Required                                        |
+| Tooltip button   | Optional                                        |
+| Sort button      | Options: none, ascending, descending            |
+| Container        | Required                                        |
+
+### Advanced Table cells
+
```

Comment:
Suggestion: it would be helpful maybe to add a sub heading to each of these cell types? Like this one would be "Parent cell" or "Expandable cell"?